main.js

```diff
@@ -16,7 +16,7 @@ async function main() {
         const options = core.getInput("options")
         const sudo    = core.getInput("sudo")
 
-        let cmd = ["ansible-playbook", playbook]
+        let cmd = [playbook]
```

Comment:
Please leave it as it was. Needs to be like that cause later we prepend sudo if needed.

Reply:
👍 resolved in bf8069946f8046c1f97e86d007633a15ad6a2e4a.